### PR TITLE
MSTest sample SDK

### PR DESCRIPTION
**Pull request by Baaryan**

### CODE REVIEW

**Review by kamal-kaur04 — CHANGES_REQUESTED** [review]
Please GitHub Actions Workflow like this - https://github.com/browserstack/xunit-browserstack/blob/main/.github/workflows/reviewing_changes.yml

**Comment by Baaryan** [comment]
Added the GitHub Actions Workflow

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,405 @@
...
+*.binlog
+
+# NVidia Nsight GPU debugger configuration file
+*.nvuser
+
+# MFractors (Xamarin productivity tool) working folder
+.mfractor/
+
+# Local History for Visual Studio
+.localhistory/
```
Comment (kamal-kaur04) [inline .gitignore]: new line

File: .github/workflows/reviewing_changes.yml [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+            repo: context.repo.repo,
+            name: process.env.job_name,
+            head_sha: process.env.commit_sha,
+            status: 'completed',
+            conclusion: process.env.conclusion
+            }).catch((err) => ({status: err.status, response: err.response}));
+            console.log(`The status-check response : ${result.status} Response : ${JSON.stringify(result.response)}`)
+            if (result.status !== 201) {
+              console.log('Failed to create check run')
+            }
```
Comment (kamal-kaur04) [inline .github/workflows/reviewing_changes.yml]: new line

File: MSTest_browserstack/SampleTest.cs [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+            string? productOnPageText = driver?.FindElement(By.XPath("//*[@id=\"1\"]/p")).Text;
+
+            driver?.FindElement(By.XPath("//*[@id=\"1\"]/div[4]")).Click();
+            bool? cartOpened = driver?.FindElement(By.XPath("//*[@class=\"float-cart__content\"]")).Displayed;
+            Assert.AreEqual(cartOpened, true);
+            string? productOnCartText = driver?.FindElement(By.XPath("//*[@id=\"__next\"]/div/div/div[2]/div[2]/div[2]/div/div[3]/p[1]")).Text;
+            Assert.AreEqual(productOnCartText, productOnPageText);
+        }
+    }
+}
```
Comment (kamal-kaur04) [inline MSTest_browserstack/SampleTest.cs]: new line

File: MSTest_browserstack/browserstack.yml [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+  # forceLocal: true  # <boolean> (Default: false) Set to true if you need to resolve all your traffic via BrowserStack Local tunnel.
+  # Entire list of arguments availabe here -> https://www.browserstack.com/docs/automate/selenium/manage-incoming-connections
+
+# ===================
+# Debugging features
+# ===================
+debug: false # <boolean> # Set to true if you need screenshots for every selenium command ran
+networkLogs: false # <boolean> Set to true to enable HAR logs capturing
+consoleLogs: errors # <string> Remote browser's console debug levels to be printed (Default: errors)
+# Available options are `disable`, `errors`, `warnings`, `info`, `verbose` (Default: errors)
```
Comment (kamal-kaur04) [inline MSTest_browserstack/browserstack.yml]: new line

**Review by kamal-kaur04 — CHANGES_REQUESTED** [review]

**Review by kamal-kaur04 — APPROVED** [review]

File: .github/workflows/reviewing_changes.yml [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+
```
Comment (kamal-kaur04) [inline .github/workflows/reviewing_changes.yml]: new line

**Review by kamal-kaur04 — CHANGES_REQUESTED** [review]

**Review by pranavj1001 — APPROVED** [review]

**Pull request merged**